run

```diff
@@ -141,7 +141,7 @@ else
     ## Determine if measurement is valid, and assign classification
     # Make measurement all lowercase
     config_measurement=`echo ${config_measurement} | tr '[:upper:]' '[:lower:]'`
-    config_classification=""
+    config_classification="\"\""
```

Comment:
Breaks [Line 160](https://github.com/flywheel-apps/mriqc/pull/29/files#diff-a53108f7543b75adbb34afc035d4cdf6R160)

I think we'd have to check right before we generate metadata.